### ISSUE TITLE
Missing file：VG-SGG-train.h5、VG-SGG-test.h5、VG-SGG-val.h5、psg-train-zs-pred、psg-val-test、psg-val-dev

### ISSUE THREAD

**Issue by jjl212** [SHTUPLUS/Pix2Grp_CVPR2024#9]
I couldn't find the file VG-SGG-train_zs_pred.h5/VG-SGG-train.h5 in the download link .Is this file the same as 'VG-SGG-with-attri.h5'? Or is VG-SGG-train_zs_pred.h5/VG-SGG-train.h5 a dataset file that has been repartitioned based on VG-SGG-with-attri.h5? I have the same confusion regarding the val and test dataset files named vg/vg_motif_anno/VG-SGG-val.h5 and vg/vg_motif_anno/VG-SGG-test.h5 in [lavis/configs/datasets/vg/defaults_rel_det_train_zs_pred.yaml](https://github.com/SHTUPLUS/Pix2Grp_CVPR2024/blob/main/lavis/configs/datasets/vg/defaults_rel_det_train.yaml). Looking forward to your response.

**Comment by Scarecrow0** [comment]
The VG annotation file is the same as the initial annotation from vg_motif_anno. The codebase will select different splits for training and testing. You can create those files through soft links.
 
The test set of our work in PSG is the official validation set of PSG. The validation set is randomly selected from the official training set by us. You can rename the official split and create your own validation set.

**Comment by Boltzmachine** [comment]
> The VG annotation file is the same as the initial annotation from vg_motif_anno. 

Sorry I did not get which file you are referring. what should `VG-SGG-train.h5` be create from?

**Comment by zhuhl0913** [comment]
> > The VG annotation file is the same as the initial annotation from vg_motif_anno.
> 
> Sorry I did not get which file you are referring. what should `VG-SGG-train.h5` be create from?

I assume that all these files (VG-SGG-train.h5, VG-SGG-test.h5, VG-SGG-val.h5) are created as soft links to the VG-SGG-with-attri.h5 file. The codebase automatically selects the appropriate data splits for further processing based on the input file name.

**Comment by Scarecrow0** [comment]
> Another file is missing during the test: zeroshot_triplet_lavis.pytorch

This file stores the triplet combination that is unseen in the training set and shares the same format with the previous zero-shot SGG codebase.
I will update this file later.

**Comment by Scarecrow0** [comment]
> > The VG annotation file is the same as the initial annotation from vg_motif_anno.
> 
> Sorry I did not get which file you are referring. what should `VG-SGG-train.h5` be create from?
 
The `VG-SGG-train.h5` is created from the initial annotation of the VG dataset: `vg_motif_anno.h5`.

**Comment by carloscaetano** [comment]
@Scarecrow0 , so, youre saying that they are all derived from VG-SGG-with-attri.h5?

To create them i should do?:

```
ln -s vg_motif_anno/VG-SGG-with-attri.h5 cache/vg/vg_motif_anno/VG-SGG-train.h5
ln -s vg_motif_anno/VG-SGG-with-attri.h5 cache/vg/vg_motif_anno/VG-SGG-test.h5
ln -s vg_motif_anno/VG-SGG-with-attri.h5 cache/vg/vg_motif_anno/VG-SGG-eval.h5
ln -s vg_motif_anno/VG-SGG-train.h5 cache/vg/vg_motif_anno/VG-SGG-train_zs_pred.h5
```

**Comment by zhaka886** [comment]
@carloscaetano  Is it possible to create files using symbolic links in this way?

**Comment by zhaka886** [comment]
@carloscaetano Is it possible to create files using symbolic links in your way?

**Comment by carloscaetano** [comment]
@zhaka886  I did not had the time to test it... :-/
